during_workshop_11.md

```diff
@@ -401,8 +398,6 @@ Follow [this tutorial](https://docs.microsoft.com/en-us/azure/storage/blobs/stor
 
 Create a new Python Azure Function App which contains a [Blob Storage Trigger](https://docs.microsoft.com/en-gb/azure/azure-functions/functions-bindings-storage-blob-trigger?tabs=python) to read in images as an input stream, when they are added to Blob Storage.
 
-Currently, Azure does not support having a Linux app and a Windows app in the same resource group so you will have to create a new resource group for your new function app. 
-

```

Comment:
Has this been fixed then, as although the [docs](https://docs.microsoft.com/en-us/azure/app-service/overview#limitations) say that there hasn't been this limitation since January, it wasn't working for me earlier this week (and [this forum post](https://feedback.azure.com/forums/169385-web-apps/suggestions/37287583-allow-a-linux-and-windows-app-service-plan-to-exis) implies they rolled it back and it wasn't working for others)?

Reply:
Ah, reverted this now. But changed to say that we will provide the resource group.